pkg/ostree/pre_run.go

```diff
@@ -0,0 +1,57 @@
+package ostree
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/openshift/microshift/pkg/backup"
+	"k8s.io/klog/v2"
+)
+
+func microshiftShouldNotRun() error {
+	cmd := exec.Command("sh", "-c", "ps aux | grep  -v grep | grep 'microshift run'")
```

Comment:
I just wanted a good method to ensure it doesn't run. We could ask systemd if you think that's enough. Also for ostree systems: if we construct service files correctly, then `pre-run` should always go before `run` so that's just a sanity check

Reply:
OK, it's good to have a check, it just seemed a little brittle to look for the command line. Let's keep this, and think about improving it in another PR.